yarn.lock

```diff
@@ -5030,7 +5030,7 @@ insert-module-globals@~6.0.0:
     through "~2.3.4"
     xtend "^3.0.0"
 
-intl-format-cache@^2.0.5:
+intl-format-cache@2.1.0, intl-format-cache@^2.0.5:
```

Comment:
it was already resolved to 2.1.0, how does this change anything?

Reply:
does this cause in changes in the yarn.lock's of consuming packages?

Reply:
Ha! I have no idea! But we consistently got failed builds last night and now we do not. Lemme try something.

Reply:
@dmfenton I just removed the `resolutions` and pushed that commit and the build failed. Then I re-added the `resolutions` and the build succeeded. I also triggered a build on master and it failed. 

So I cannot explain exactly whey this works but I am confident that it does.

Reply:
> does this cause in changes in the yarn.lock's of consuming packages?

I would not expect it too